anchore_engine/analyzers/manager.py

```diff
@@ -56,9 +57,8 @@ def _run_analyzer_modules(analyzer_report, configdir, imageId, unpackdir, output
                     )
                 else:
                     logger.debug(
-                        "command succeeded: cmd=%s stdout=%s stderr=%s".format(
-                            repr(cmdstr), repr(sout.strip()), repr(serr.strip())
-                        )
+                        "command succeeded: cmd=%s stdout=%s stderr=%s"
+                        % (repr(cmdstr), repr(sout.strip()), repr(serr.strip()))
```

Comment:
if you are changing this already, it is safer to skip the `%` altogether and use the rest as arguments (see https://github.com/anchore/enterprise/issues/150 for reference):

```suggestion
                        "command succeeded: cmd=%s stdout=%s stderr=%s",
                        repr(cmdstr), repr(sout.strip()), repr(serr.strip())
```

Reply:
got it, fixed